mosviz/viewers/mos_viewer.py

```diff
@@ -882,7 +882,7 @@ def _load_comments(self, label):
 
         #Fill in any saved comments:
         meta = data.meta
-        obj_names = data.get_component("id")._categorical_data
+        obj_names = data.get_component("id").labels
```

Comment:
@astrofrog Is there ever a chance that there is no component called `id`? I think it would return `None` and then start throwing errors.  Would it be possible to either do a little error checking here or to add a `get_labels("id")` kind of thing to the data object (class)?  

(same comment for ones below)